website/recursion_website/templates/recursion_website/base.html

```diff
@@ -0,0 +1,59 @@
+<!Doctype html>
+<html>
+<head>
+	<title>RECursion</title>
+	<!--Package Imports-->
+
+	<!-- Compiled and minified MATERIAL CSS -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <!-- Compiled and minified MATERIAL JavaScript -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+
+    <link rel="stylesheet" href="base_css.css">

```

Comment:
Instead of this you can write like, `<link rel="stylesheet" href="{% static 'css/base_css.css' %}">`

now we can directly access `base.html` from anywhere